core/src/test/resources/HistoryServerExpectations/stage_blacklisting_for_stage_expectation.json

```diff
@@ -0,0 +1,639 @@
+{
```

Comment:
nit: "stage" twice in the filename is confusing, how about just "blacklisting_for_stage_expectation.json"